rocAL/rocAL/include/api/rocal_api_data_loaders.h

```diff
@@ -705,8 +583,7 @@ extern "C"  RocalStatus  ROCAL_API_CALL rocalResetLoaders(RocalContext context);
 /// \param decode_size_policy
 /// \param max_width The maximum width of the decoded images, larger or smaller will be resized to closest
 /// \param max_height The maximum height of the decoded images, larger or smaller will be resized to closest
-/// \return Reference to the output image
-extern "C"  RocalImage  ROCAL_API_CALL rocalJpegCaffeLMDBRecordSourcePartialSingleShard(RocalContext p_context,
+extern "C"  RocalTensor  ROCAL_API_CALL rocalJpegCaffeLMDBRecordSourcePartialSingleShard(RocalContext p_context,
```

Comment:
Move these API's before reset loaders.

Reply:
In the open source rocal_api_dataloaders.h file the order is like this hence retaining it here